boards/arduino-mega2560/src/pins.rs

```diff
@@ -0,0 +1,346 @@
+        pub d50: portb::pb3::PB3,
+        /// `D51`
+        ///
+        /// * `MOSI`: SPI bus Master Out/Slave In
+        /// * `PCINT2`: External Interrupt (Pin Change)
+        pub d51: portb::pb2::PB2,
+        /// `D52`
+        ///
+        /// * `SCK`: SPI bus Serial Clock
+        /// * `PCINT1`: External Interrupt (Pin Change)
+        pub d52: portb::pb1::PB1,
+        /// `D53`
+        ///
+        /// * `SS`: SPI bus Chip Select
+        /// * `PCINT0`: External Interrupt (Pin Change)
+        pub d53: portb::pb0::PB0,
+        /// `A0`
+        ///
+        /// * `ADC0`: A/D converter input 0
+        pub a0: portf::pf0::PF0,
+        /// `A1`
+        ///
+        /// * `ADC1`: A/D converter input 1
+        pub a1: portf::pf1::PF1,
+        /// `A2`
+        ///
+        /// * `ADC2`: A/D converter input 2
+        pub a2: portf::pf2::PF2,
+        /// `A3`
+        ///
+        /// * `ADC3`: A/D converter input 3
+        pub a3: portf::pf3::PF3,
+        /// `A4`
+        ///
+        /// * `ADC4`: A/D converter input 4
+        /// * `TCK`: JTAG test clock
+        pub a4: portf::pf4::PF4,
+        /// `A5`
+        ///
+        /// * `ADC5`: A/D converter input 5
+        /// * `TMS`: JTAG test mode select
+        pub a5: portf::pf5::PF5,
+        /// `A6`
+        ///
+        /// * `ADC6`: A/D converter input 6
+        /// * `TDO`: JTAG test data output
+        pub a6: portf::pf6::PF6,
+        /// `A7`
+        ///
+        /// * `ADC7`: A/D converter input 7
+        /// * `TDI`: JTAG test data input
+        pub a7: portf::pf7::PF7,
+        /// `A8`
+        ///
+        /// * `ADC8`: A/D converter input 8
+        /// * `PCINT16`: External Interrupt (Pin Change)
+        pub a8: portk::pk0::PK0,
+        /// `A9`
+        ///
+        /// * `ADC9`: A/D converter input 9
+        /// * `PCINT17`: External Interrupt (Pin Change)
+        pub a9: portk::pk1::PK1,
+        /// `A10`
+        ///
+        /// * `ADC10`: A/D converter input 10
+        /// * `PCINT18`: External Interrupt (Pin Change)
+        pub a10: portk::pk2::PK2,
+        /// `A11`
+        ///
+        /// * `ADC11`: A/D converter input 11
+        /// * `PCINT19`: External Interrupt (Pin Change)
+        pub a11: portk::pk3::PK3,
+        /// `A12`
+        ///
+        /// * `ADC12`: A/D converter input 12
+        /// * `PCINT20`: External Interrupt (Pin Change)
+        pub a12: portk::pk4::PK4,
+        /// `A13`
+        ///
+        /// * `ADC13`: A/D converter input 13
+        /// * `PCINT21`: External Interrupt (Pin Change)
+        pub a13: portk::pk5::PK5,
+        /// `A14`
+        ///
+        /// * `ADC14`: A/D converter input 14
+        /// * `PCINT22`: External Interrupt (Pin Change)
+        pub a14: portk::pk6::PK6,
+        /// `A15`
+        ///
+        /// * `ADC15`: A/D converter input 15
+        /// * `PCINT23`: External Interrupt (Pin Change)
+        pub a15: portk::pk7::PK7,
+    }
```

Comment:
I'd additionally add `led_tx` (= `PD5`) and `led_rx` (= `PD4`) as well.

Reply:
Those LEDs are connected to port D on the ATmega32U4 that manages the USB interface, not port D of the ATmega2560.

Reply:
Ah, I see!